plugins/source/aws/codegen/main.go

```diff
@@ -7,55 +7,56 @@ import (
 )
 
 func main() {
-	resources := make([]*recipes.Resource, 0, 1000)
-	resources = append(resources, recipes.AccessAnalyzerResources()...)
+	var resources []*recipes.Resource
 	resources = append(resources, recipes.ACMResources()...)
 	resources = append(resources, recipes.APIGatewayResources()...)
 	resources = append(resources, recipes.APIGatewayV2Resources()...)
+	resources = append(resources, recipes.AccessAnalyzerResources()...)
+	resources = append(resources, recipes.AppSync()...)
 	resources = append(resources, recipes.ApplicationAutoScalingResources()...)
 	resources = append(resources, recipes.ApprunnerResources()...)
-	resources = append(resources, recipes.AppSync()...)
 	resources = append(resources, recipes.AthenaResources()...)
 	resources = append(resources, recipes.AutoscalingResources()...)
 	resources = append(resources, recipes.BackupResources()...)
+	resources = append(resources, recipes.CloudHSMV2()...)
+	resources = append(resources, recipes.CloudWatchLogsResources()...)
 	resources = append(resources, recipes.CloudformationResources()...)
 	resources = append(resources, recipes.CloudfrontResources()...)
-	resources = append(resources, recipes.CloudHSMV2()...)
 	resources = append(resources, recipes.CloudtrailResources()...)
-	resources = append(resources, recipes.CloudWatchLogsResources()...)
 	resources = append(resources, recipes.CloudwatchResources()...)
 	resources = append(resources, recipes.CodeBuildResources()...)
 	resources = append(resources, recipes.CodePipelineResources()...)
 	resources = append(resources, recipes.CognitoResources()...)
 	resources = append(resources, recipes.ConfigResources()...)
+	resources = append(resources, recipes.DMSResources()...)
 	resources = append(resources, recipes.DaxResources()...)
 	resources = append(resources, recipes.DirectConnectResources()...)
-	resources = append(resources, recipes.DMSResources()...)
 	resources = append(resources, recipes.DynamoDBResources()...)
 	resources = append(resources, recipes.EC2Resources()...)
 	resources = append(resources, recipes.ECRPublicResources()...)
 	resources = append(resources, recipes.ECRResources()...)
 	resources = append(resources, recipes.ECSResources()...)
 	resources = append(resources, recipes.EFSResources()...)
 	resources = append(resources, recipes.EKSResources()...)
-	resources = append(resources, recipes.ElastiCacheResources()...)
-	resources = append(resources, recipes.ElasticbeanstalkResources()...)
-	resources = append(resources, recipes.ElasticsearchResources()...)
 	resources = append(resources, recipes.ELBv1Resources()...)
 	resources = append(resources, recipes.ELBv2Resources()...)
 	resources = append(resources, recipes.EMRResources()...)
+	resources = append(resources, recipes.ElastiCacheResources()...)
+	resources = append(resources, recipes.ElasticbeanstalkResources()...)
+	resources = append(resources, recipes.ElasticsearchResources()...)
 	resources = append(resources, recipes.EventbridgeResources()...)
```

Comment:
Can you keep this list sorted alphabetically?

Reply:
reverted sort